ieeeuoft/hackathon-template
# fixed react warnings and added feature flags for rsvp and teams

## Overview

- Resolves: N/A
- fixed react warnings (the ones that appear on npm run build)
- added feature flags for teams and rsvp (this will help us when removing/adding features)
- added npm run build to ci checks to catch warnings on all branches


## Unit Tests Created

- edited tests that have to do with teams and rsvp


## Steps to QA

- disable teams in settings/__init__.py and ensure that anything to do with teams doesn't appear on localhost:8000
- don't need to test RSVP because it requires a migration, we'll test it out when we create the MakeUofT repo
